pkg/apis/core/v1alpha1/types_shoot.go

```diff
@@ -744,6 +744,8 @@ type Alerting struct {
 // Provider contains provider-specific information that are handed-over to the provider-specific
 // extension controller.
 type Provider struct {
+	// EnableContainerD indicate if kubelet should be started with ContainerD and not docker
+	EnableContainerD bool  `json:"enableContainerD,omitempty"`
```

Comment:
Same like for v1beta1 version

Reply:
Please have a look on my pkg/apis/core/v1beta1/types_shoot.go comment.
Once we will decide i will update also for v1alpha1